scania-digital-design-system/tegel
# fix(card): toggle stretch mode

## **Describe pull-request**  
Scaling cards

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3445-scaling-cards](https://tegel.atlassian.net/browse/CDEP-3445)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to storybook for Card
2. Toggle stretch on/off and see how the card changes

## **Checklist before submission**

## **Suggested test steps**

## **Additional context**  
Unit test might be added later
